### PR TITLE
Update China's public holidays for 2022.

### PR DESCRIPTION
**Pull request by iamsk**
Update China's public holidays for 2022.

refs #

http://www.gov.cn/zhengce/content/2021-10/25/content_5644835.htm

### CODE REVIEW

**Comment by iamsk** [comment]
@brunobord Please have a look, thanks!

File: workalendar/asia/china.py [hunk 1]
```diff
@@ -66,13 +73,21 @@
...
+            'Labour Day Holiday Shift': [(4, 24), (5, 7)],
+            'National Day Shift': [(10, 8), (10, 9)]
+        },
 }
 
 
 @iso_register('CN')
 class China(ChineseNewYearCalendar):
     "China"
     # WARNING: Support 2018, 2019 currently, need update every year.
```
Comment (brunobord) [inline workalendar/asia/china.py:89]: we may also want to change this comment, don't you think.

**Review by brunobord — APPROVED** [review]
I'll approve like this, even if there's a tiny outdated comment.
If you agree, I'll merge this tomorrow.

File: workalendar/asia/china.py (hunk 1 above)
Comment (iamsk) [reply workalendar/asia/china.py]: updated, thanks for your comment.

**Comment by brunobord** [comment]
The latest version of workalendar now includes your very valuable patch.

* https://github.com/workalendar/workalendar/releases/tag/16.2.0
* https://pypi.org/project/workalendar/16.2.0/

Happy upgrading!

**Pull request merged**